docs/_user_guide/best_practices_matlab_slurm_job.md

```diff
@@ -1,33 +1,33 @@
 ## Matlab Example Slurm Script
 The simplest Matlab script looks like:
 
-```matlab
+```matlab title="Matlab Code"
 disp('Hello world')
 ```
 
 Save this line to a new file called `hello_world.m`. 
 
-## Available Matlab versions on Yen10
+## Available Matlab versions on `Yen10`
 Several version of Matlab are installed on the yens and the default version is designated with `(D)`:
 
-```bash
+```bash title="Terminal Command"
 module avail matlab
 ```
 
 Load the version you want:
 
-```bash
+```bash title="Terminal Command"
 module load matlab
```

Comment:
change this example to be one of the non-default versions like `module load matlab/R2021b`